types/src/front/assistant/conversation.ts

```diff
@@ -127,6 +127,8 @@ export type AgentMessageStatus =
   | "failed"
   | "cancelled";
 
+export type AgentMessageFeedbackDirection = "up" | "down";
```

Comment:
Let's not add those to `types` as we plan to remove them. You can create a file in `lib/api`.